frontend/src/components/FormInput/forminputElementRadioButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './forminputElementRadioButton.css';
+
+export function RadioButtonFormInputFifthExamComponent() {
+  return (
+    <div className="radioButtonInputFormElement">
+      <label htmlFor="theInput">
+        <div className="labelTextDiv">Präsentationsprüfung:</div>
+        <div className="inputDivRadioElement"><input type="radio" id="theInput" className="radioButtonInput" /></div>
+      </label>
+    </div>
+  );
+}
+export default RadioButtonFormInputFifthExamComponent;
```

Comment:
wenn du die KOmponente hier als default exportierst, brauchst du sie oben in Zeile 4 nicht auch exportieren. Also einfach am besten das 'export' aus Zeile 4 entfernen

Reply:
Gleiche Sache auch an anderen Stellen

Reply:
Ok nehme ich raus